atk-sys-test/build.rs

```diff
@@ -0,0 +1,60 @@
+extern crate ctest;
+extern crate shell_words;
+
+fn pkg_config_cflags() -> Result<Vec<String>, Box<std::error::Error>> {
+    let mut cmd = std::process::Command::new("pkg-config");
+    cmd.arg("--cflags");
+    cmd.arg("atk");
+    let out = cmd.output()?;
+    if !out.status.success() {
+        return Err(format!("command {:?} returned {}",
+                           &cmd, out.status).into());
+    }
+    let stdout = std::str::from_utf8(&out.stdout)?;
+    Ok(shell_words::split(stdout.trim())?)
```

Comment:
Is it required to send back `Ok()`? Wouldn't just `shell_words::split(stdout.trim())` work?

Reply:
Different error type.